relay-cardinality/src/redis/limiter.rs

```diff
@@ -233,6 +236,24 @@ impl IntoIterator for CheckedLimits {
     }
 }
 
+/// Buckets the amount of scopes contained in a scope into a metric tag.
+fn num_scopes_tag(state: &LimitState<'_>) -> &'static str {
+    match state.scopes().len() {
+        0 => "0",
+        1 => "1",
+        2 => "2",
+        3 => "3",
+        4 => "4",
+        5 => "5",
+        ..=10 => "10",
```

Comment:
Should we add a prefix to the numbers?
```suggestion
        ..=10 => "<=10",
```

Reply:
Not a fan, I think it generates more noise in the metric and `<= 10` is still technically incorrect it would need to be `5 < x <= 10` which becomes really long and unreadable.